### PR TITLE
WandX-OptionsDapp

### PR DESCRIPTION
**Pull request by bssrdev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,50 @@
...
+## Getting Started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+What things you need to install the software and how to install them
+
+```
+1. Node js
+2. Node Package Manager
+3. Angular 2 or 4 or 5
```
Comment (satyamakgec) [inline README.md]: Angular v2 or above

File: README.md [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+4. Mongo DB to run the local database
+5. Code Editor and Command Prompt
```
Comment (satyamakgec) [inline README.md]: No need to mention 5 point

File: README.md [hunk 3]
```diff
@@ -1,2 +1,50 @@
...
+> Setting Up the Backend Environment
+
+1. Download this github project.
+2. Open WandXOptionsDapp
+3. Run the below commands in the Command Window by moving to the exact location where the project is inside the terminal.
+```
+> npm install
+> npm package.json
+> npm nodemon install
+> nodemon
```
Comment (satyamakgec) [inline README.md]: only two commands needed I think
npm install 
npm install -g nodemon

**Review by satyamakgec — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bssrdev) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (bssrdev) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (bssrdev) [reply README.md]: Done

**Review by satyamakgec — APPROVED** [review]

**Review by SatyamSB — APPROVED** [review]

**Pull request merged**